spec/ddtrace/configuration/pin_setup_spec.rb

```diff
@@ -42,8 +42,8 @@
     end
   end
 
-  describe 'Datadog#configure' do
-    before { Datadog.configure_onto(target, service_name: :foo, extra: :bar) }
+  describe 'Datadog::Tracing#configure' do
+    before { Datadog::Tracing.configure_onto(target, service_name: :foo, extra: :bar) }
 
     it do
       expect(target.datadog_pin.service).to eq(:foo)
```

Comment:
Is this still needed? We already have a spec for this in the `tracing-spec.rb`

Reply:
Yup, indeed it was already like this. My question is, since we're touching it, and given the current state of the code, I don't think this test makes sense, because it's testing the `Tracing` class, and we already have tests for that.